dev-libs/gobject-introspection/gobject_introspection-1.55.0.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="Introspection system for GObject-based libraries"
+DESCRIPTION="GObject introspection is a middleware layer between C libraries \
+(using GObject) and language bindings. The C library can be scanned at \
+compile time and generate a metadata file, in addition to the actual native \
+C library. Then at runtime, language bindings can read this metadata and \
+automatically provide bindings to call into the C library."
+HOMEPAGE="https://wiki.gnome.org/Projects/GObjectIntrospection"
+COPYRIGHT="Colin Walters
+	Johan Dahlin
+	Matthias Clasen (girepository)
+	Jürg Billeter (scanner)"
+LICENSE="GNU LGPL v2
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://ftp.gnome.org/pub/GNOME/sources/gobject-introspection/1.55/gobject-introspection-$portVersion.tar.xz"
+CHECKSUM_SHA256="888fba5df3ebfa42ea9f24ad98a667fdc151cf37a07349352804028a5ef0948f"
+SOURCE_DIR="gobject-introspection-$portVersion"
+PATCHES="gobject_introspection-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gobject_introspection$secondaryArchSuffix = $portVersion compat >= 1
```

Comment:
Maybe this would be a good idea.
```diff
PROVIDES="
-	gobject_introspection$secondaryArchSuffix = $portVersion compat >= 1
+	gobject_introspection$secondaryArchSuffix = $portVersion
```
@alaviss, what do you think?